cmd/join.go

```diff
@@ -291,3 +304,19 @@ func createVersionStr(k3sVersion, k3sChannel string) string {
 	}
 	return installStr
 }
+
+func makeJoinExec(serverIP, joinToken, installStr, k3sExtraArgs string, serverAgent bool) string {
```

Comment:
Can you build the whole string by using a slice of strings, and at the end combine them with `strings.Join`? That would make the approach easier to read/maintain over time and more consistent with the install command.

Reply:
@alexellis you mean put each string to it's own variable and join them at the end?